build_pdf4qt.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+set -e
+
+# Install VCPKG
+git clone https://github.com/microsoft/vcpkg.git
+cd vcpkg
+./bootstrap-vcpkg.sh -disableMetrics
+cd ..
+export VCPKG_ROOT=$(pwd)/vcpkg
+
+# Install tbb
+${VCPKG_ROOT}/vcpkg install tbb
+
+# Build
+cd pdf4qt
+export CMAKE_BUILD_PARALLEL_LEVEL=$(nproc)
```

Comment:
it's NEVER nproc. We have `FLATPAK_BUILDER_N_JOBS` for that. Using nproc make it impossible to set the number of parallel task with flatpak-builder.

See my comment about using `buildsystem: cmake-ninja`

Reply:
Removed during update to cmake-ninja.